docs/specification/1.0.md

```diff
@@ -523,6 +524,36 @@ The following example illustrates a delete action:
 }
 ```
 
+
+
+#### overrideReason
```

Comment:
Suggest this should be capitalized `OverrideReason` since it is the name of the type